src/include/daos_srv/vos_types.h

```diff
@@ -319,6 +319,8 @@ typedef struct {
 	daos_key_t		ip_dkey;
 	/** attribute key (VOS_ITER_DKEY/RECX/SINGLE, standalone only) */
 	daos_key_t		ip_akey;
+	/** address range (RECX); 0:0 is is entire range (0:~0ULL) */
```

Comment:
typo "is is" ?  Also, should this be same as comment in vos_internal (e.g. rx_nr == 0 means entire range)

Reply:
Fixed.